PicoProducer/test/testSFs.py

```diff
@@ -15,7 +15,7 @@
 maxerr     = 10 # maximum number of errors
 neta       = True # include negative eta
 ptvals_    = [ 10., 20., 21., 22., 24., 25., 26., 27., 34., 35., 36., 40., 60., 156., 223., 410., 560. ]
-etavals_   = [ 0.0, 0.5, 1.1, 1.9, 2.3, 2.4, 2.8, 3.4 ]
+etavals_   = [ 0.0, 0.5, 1.1] #, 1.9, 2.3, 2.4, 2.8, 3.4 ]
```

Comment:
Why did you reduce this list? It's to check how the tool handles errors.

Reply:
@IzaakWN I should remove this change, it was just convenient for me during a test